### ISSUE TITLE
Stoploss is not triggered while there is a not filled sell order

### ISSUE THREAD

**Issue by gathad** [freqtrade/freqtrade#3564]
## Describe your environment

  * Operating system: WSL2 Ubuntu 20.04 LTS
  * Python Version: python 3.8.2
  * CCXT version: 1.30.34
  * Freqtrade Version: freqtrade developer-11290645
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem: When roi sell is activated, stoploss is not working anymore.

*Explain the problem you have encountered*
I'm sending sell signals from order books ask side. And if the sell signal does not filled, and market turns back and reach the stoploss level, stoploss is not reacting.

### Steps to reproduce:

  1. Set the ask strategy to use order books ask side.
  2. Set minimal roi as "0: -1" to send an sell signal right after opening the trade.
  3. Set stoploss to a reachable level before minimal roi.
  4. After sell signal because of roi is sended, wait until price goes lower than stoploss.

  
### Observed Results:

* What happened?
Stoploss is not triggered, price goes way more lower and bot waits for filling of the first sell signal. After timeout, it sends a stoploss sell signal with the current prices order books first ask level.

  * What did you expect to happen?
I was expecting that when stoploss level is reached, stoploss cancels the current sale order which is no filled and send a new sell order because of stoploss.

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
  // paste your log here
  ```

**Comment by gathad** [comment]
Hello,

I have found this today while I'm searching for a solution: #2872

Is sell_check_timeout working properly right now? If yes, I can use it to cancel my unfilled sell orders to send a stoploss order.

**Comment by xmatthias** [comment]
i'm not aware that `sell_check_timeout()` would've been not working at any point in the past since it's been added.
so yes, in theory you should be able to use this as a workaround for the problem you're describing.

You could also simply lower the `unfilledtimeout.sell` so the order isn't there for a long time, but just for a minute or two.

---
 

I am however curious about the values used (for roi / stoploss) - as it sounds a bit like a "way too close stoploss" problem to me.

**Comment by gathad** [comment]
Hello,

I wasn't able to write here again because of so much work. I'm working on low priced cryptos. So, stoploss is less than 1 price step. Minimal roi is -1 so, it's sending a sell signal just after buying a crypto (bought from bid price sell from ask price) and if price go 1 step lower, trying to sell from ask price again which I have bought. Thats my strategy for low risk/high profit. But also comes with high possibility to sell from bought price. 

Since this is my strategy, market volumes are not high as other cryptos. And that's why I can't lower the unfilledtimeout.sell. But if I can cancel the order with another logic with sell_check_timeout() it's not a problem. I can check the bids and asks and I can sell according to this information.

So, I have solved that problem right now. It's working like my desire. But I still have another problem about time sync and I have used bot_loop_start to solve that. But it's not working properly. My time shifts about 1 seconds in 6 hours. So, I have write a sh command with subprocess (checked on another python file, works fine) but it's not working automaticly while freqtrade is working. Freqtrade is not giving any error. But my time is still shifting 1 second in 6-6,5 hours and can't communicate with binance again.
